src/website/views/components/alert.pug

```diff
@@ -99,29 +113,56 @@ p.-text
     each type in ['success', 'danger', 'warning', 'info', 'light', 'dark']
       .m-alert.-toast.-hasIcon.-m2(class=`-${type}`, data-title=`${type}`)
         = `This is a ${type} alert`
-        button(aria-label="Close")
+        button(class='a-btn -icon -close')
+          .a-btn__content
+            .a-icon
+              i.a-icon.icon-x
```

Comment:
All instances of `.a-icon.icon-x` within **Alerts** should also have an `.-xs` size class to ensure the icon renders at 12x12.